src/inverseRobot.js

```diff
@@ -24,7 +24,19 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const object = {};
```

Comment:
it would be better to use more clear naming here.
```suggestion
  const fixedRobot = {};
```